tests/test_density_CorrelationFunction.py

```diff
@@ -455,13 +456,12 @@ def value_func(_r):
         for nq, neighbors in test_set:
```

Comment:
In this case, parametrizing over `rv` will require the test to re-generate the test set and compute the `supposed_correlation` multiple times. I think that might be a nontrivial computation and would potentially increase test time. It may be possible to use a fixture that is cached to avoid recomputing that data?